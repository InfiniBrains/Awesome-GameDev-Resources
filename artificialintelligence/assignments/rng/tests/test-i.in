1234567 50 123 321
