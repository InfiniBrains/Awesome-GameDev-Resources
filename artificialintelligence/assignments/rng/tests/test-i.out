136
190
282
319
176
310
192
244
321
298
292
239
274
197
123
210
282
319
149
214
244
197
198
144
310
306
165
256
166
213
229
156
283
168
279
129
137
251
310
249
140
128
184
175
159
273
160
172
258
263
